src/main/scala/com/gu/contentapi/utils/PredicateUtils.scala

```diff
@@ -0,0 +1,20 @@
+package com.gu.contentapi.utils
+
+object PredicateUtils {
+
+  implicit class RichPredicate[A](val f: Function1[A, Boolean]) extends AnyVal {
```

Comment:
you can find in the main Lambda class the following code:

```scala
import com.gu.contentapi.utils.PredicateUtils._
  

val downloadsLogs = allAcastLogs.filter(AcastLog.onlyDownloads && AcastLog.onlySuccessfulReponses)
```

This enable boolean logic operators on boolean function.